cetmix_tower_aws/readme/DESCRIPTION.md

```
@@ -0,0 +1,6 @@
+**⚠️ DISCLAIMER: TECHNICAL MODULE**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)